src/libcore/unicode/unicode.py

```diff
@@ -21,11 +21,14 @@
 # - UnicodeData.txt
 #
 # Since this should not require frequent updates, we just store this
-# out-of-line and check the unicode.rs file into git.
+# out-of-line and check the unicode.py file into git.
```

Comment:
I’m pretty sure that the original comment did mean `.rs`, even though the generated file has since been renamed a couple times (now `tables.rs`).

Reply:
Ah, I thought it could have been either. I've updated it to `tables.rs`.